pkg/network/state.go

```diff
@@ -248,6 +252,8 @@ func (ns *networkState) StoreClosedConnection(conn ConnectionStats) {
 			prev.MonotonicSentBytes += conn.MonotonicSentBytes
 			prev.MonotonicRecvBytes += conn.MonotonicRecvBytes
 			prev.MonotonicRetransmits += conn.MonotonicRetransmits
+			prev.MonotonicTCPEstablished += conn.MonotonicTCPEstablished
+			prev.MonotonicTCPClosed += conn.MonotonicTCPClosed
```

Comment:
Perhaps we can simplify the state machine logic and throw some of this addition/subtraction logic in separate functions?

Reply:
Totally agreed, I think there is a the state bits are needing some refactor. I'll create a Jira card assign to myself if you don't mind, since I think this will require some time investment.